README.md

```diff
@@ -168,11 +168,11 @@ The listener object created may emit the following events:
 
 ### Start a listener
 
-- `JavaScript` - `await listener.listen(multiaddr)`
+- `JavaScript` - `await listener.listen(Array<multiaddr>)`
```

Comment:
What happens if it fails to bind to one or more of the addresses?

Reply:
Ideally this would eventually be configurable, https://github.com/libp2p/js-libp2p-websocket-star/issues/61#issuecomment-437425776, so you could specify exactly what happens.

The default behavior we decided would be good is to only error if **all** addresses fail. This is similar to how websocket-star-multi behaves today.

I think a subsequent improvement would be to add the configuration option. We could add them now, but I wanted to try and block the async changes as little as possible.